### PR TITLE
Add admin emails to the mirror status JSON output when logged in

### PR DESCRIPTION
**Pull request by kyrias**

### CODE REVIEW

File: mirrors/views/api.py [hunk 1]
```diff
@@ -92,6 +92,9 @@ def mirror_details_json(request, name):
             show_all=authorized)
     data = status_info.copy()
     data['version'] = 3
+    if authorized:
+        data['admin_email'] = mirror.admin_email
+        data['alternate_email'] = mirror.alternate_email
```
Comment (jelly) [inline mirrors/views/api.py]: Do we want to restrict this to only specific roles? @Bluewind

Comment (Bluewind) [reply mirrors/views/api.py]: That would be better, yes. Can we restrict it the same way the email is restricted on the html version?

Comment (jelly) [reply mirrors/views/api.py]: We can probably restrict it on either staff groups or user permissions

Comment (Bluewind) [reply mirrors/views/api.py]: Looks like we currently make this public to all logged in users[1]. I'd prefer having it check for the mirrors.change_mirror permission, but it's up to you. 

[1] https://github.com/archlinux/archweb/blob/master/templates/mirrors/mirror_details.html#L35

Comment (kyrias) [reply mirrors/views/api.py]: Done

**Comment by Bluewind** [comment]
Good from my side

**Comment by jelly** [comment]
Thanks for the PR, merged :)

**Pull request merged**